### PR TITLE
 AZ Input files for ephx2 ptp1b - charge_annihilation

### PR DESCRIPTION
**Pull request by benefrg**
Added files via upload.

## Input submission checklist

* [ok] created a directory for each system based on the [template directory][templates]
* [ok] named and placed the directory with the following pattern: `input_structures/prepared_structures/<set_name>/<system_name>`

For each submitted directory:

* [ok] filled in system preparation details in the `PREPARATION_DETAILS.md` file
* [ok] added a PDB file with the protein named `protein.pdb`
* [ok] removed any cofactors from the PDB file and placed them in `cofactors.sdf`
* [ok] did not re-protonate the residues, but left protonation state of the protein unchanged
* [ok] kept cystallographic waters and metals in the PDB file
* [ok] tested the PDB and, if available, cofactors.sdf file using the [input validation script][input_validation]
* [ok] copied over the ligands SDF file to a file named `ligands.sdf` and removed any duplicate binding modes or protonation states, keeping the more favorable state

## Licensing agreement
* [ ] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/ephx2/ligands.sdf
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/ephx2/ligands.sdf:1]: There are multiple protonation states for ligands 19 and 47 in this file, you will need to keep only one of them, see: https://industrybenchmarks2024.readthedocs.io/en/latest/public/input_preparation.html#preparing-the-ligand-file

Looking at the original Chen et al. paper (https://pubs.acs.org/doi/full/10.1021/acs.jctc.8b00825), the set anticipates 2 charge transformations. So you should retain 47charg and 19charg, and remove 47neu and 19neu.

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/ephx2/PREPARATION_DETAILS.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+# Protein preparation
+* ACE and NMA caps were added using the "build" function in Schrodinger's Maestro (Release 2023-4)
+	- Final residue numbers: ACE236 and NMA549
+* NMA was manually renamed to NME
+* Hydrogen names of ACE and NME were manually altered 
+* The protein.pdb passed the validation script (python input_validation.py --pdb protein.pdb)
+
+# Ligand preparation
+* All ligands were used without any additional modification
```
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/ephx2/PREPARATION_DETAILS.md:15]: Please update this based on the ligands you end up selecting.

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/ptp1b/ligands.sdf
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/ptp1b/ligands.sdf:1]: Similarly to EPHX2, only a single protonation state per ligand should be kept. In this case, it looks like 26neu should be removed.

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/ptp1b/protein.pdb
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/ptp1b/protein.pdb:1]: In the original schrodinger PDB, the C and N termini were both charges / uncapped. As per https://industrybenchmarks2024.readthedocs.io/en/latest/public/input_preparation.html#capping-proteins (see "if the termini are charged"), we should aim to keep these uncapped, unless there is a good reason to not do so.

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/ephx2/protein.pdb
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/ephx2/protein.pdb:1]: This PDB looks great thanks!

**Review by IAlibay — CHANGES_REQUESTED** [review]
Thanks for submitting this @benefrg.

Two main changes required:
1. A single protonation state should be kept for the ligands, I've annotated which ones I think should be kept.
2. The PTP1B protein doesn't look like it should be capped.

Let me know if there's any extra information / help I can provide!

**Comment by benefrg** [comment]
Thanks for reviewing this @IAlibay

I updated and uploaded the modified files. Please let me know if further modifications are needed.

**Review by IAlibay — APPROVED** [review]
Thanks @benefrg !

**Pull request merged**